lib/cylc/config.py

```diff
@@ -955,7 +957,7 @@ def compute_inheritance(self, use_simple_method=True):
             OUT.info("Parsing the runtime namespace hierarchy")
 
         results = OrderedDictWithDefaults()
-        n_reps = 0
+        # n_reps = 0
```

Comment:
(I think these were being kept to support a commented-out comparison between alternative methods, that we should look at some time...)